icu4c/source/test/intltest/alphaindextst.cpp

```diff
@@ -756,4 +757,52 @@ void AlphabeticIndexTest::checkHasBuckets(const Locale &locale, UScriptCode scri
             uscript_getScript(bucket->getLabel().char32At(0), errorCode));
 }
 
+void AlphabeticIndexTest::checkMaxLabelCount() {
+    IcuTestErrorCode errorCode(*this, "checkMaxLabelCount");
+    UErrorCode status = U_ZERO_ERROR;
```

Comment:
Why do you have both `errorCode` and `status`? Just use the higher-function `errorCode`.